site/src/components/layout.md

```diff
@@ -136,6 +140,76 @@ The specification for this layout is as follows.
 | Desktop    | Menu, main, aside are side by side. Main is wider than menu.                                                                  |
 | Widescreen | Same as desktop with scaling gutter that keeps combined width of menu, main, aside to the width of the widescreen breakpoint. |
 
+### Sidecar left layout
+
+A "sidecar" is a smaller companion container that sits beside the main content container on tablet screens and wider. In `layout-sidecar-left` the sidecar refers to the `layout-body-menu` element, which sits to the left of `layout-body-main`. Unlike other layouts, this layout does not allow the usage of the `layout-body-aside` containers.
+
+How do I apply it? `layout-sidecar-left` to the `layout` element.
```

Comment:
Can we add a new line between these three sentences like in the single layout? It seems hard to parse the required elements and allowed elements, especially for sidecar-left/right

![image](https://github.com/microsoft/atlas-design/assets/94572161/536c6d5a-9fed-435e-8714-1cf8d2aff4da)

Reply:
Weird, guess the single newline is collapsed. I'll update all the ones after single.

Reply:
Updated